fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java

```diff
@@ -96,75 +90,16 @@ private void writeUnlock() {
     @Override
     public synchronized void start() {
         super.start();
-        exportingExecutor.start();
     }
 
     @Override
     protected void runAfterCatalogReady() {
```

Comment:
We don't need to extends from MasterDaemon any more

Reply:
done